src/packet/h264.rs

```diff
@@ -208,7 +208,7 @@ impl Depacketizer for H264Depacketizer {
         out: &mut Vec<u8>,
         _: &mut CodecExtra,
     ) -> Result<(), PacketError> {
-        if packet.len() <= 2 {
+        if packet.len() == 0 {
```

Comment:
Why can this be zero?

Reply:
It can't ;)

Reply:
Could write this condition as `packet.len() < 1` if you want

Reply:
Right. So what was wrong with <= 2? :)

Reply:
It errors on payloads of length 1 and 2, which are allowed

Reply:
I copied this code from webrtc-rs – they would have the bug too. Maybe we should upstream?

Reply:
Oh. I missed that. Thanks.